products.json

```diff
@@ -1699,9 +1699,449 @@
+          "chain": false,
+          "matrix": false,
+          "infrared": false,
+          "multizone": false,
+          "temperature_range": [
+            1500,
+            9000
+          ]
+        },
+        "upgrades": []
+      },
+      {
+        "pid": 185,
+        "name": "LIFX Candle Color US",
+        "features": {
+          "color": true,
+          "chain": false,
+          "matrix": true,
+          "infrared": false,
+          "multizone": false,
+          "temperature_range": [
+            1500,
+            9000
+          ]
+        },
+        "upgrades": []
+      },
+      {
+        "pid": 186,
+        "name": "LIFX Candle Color Intl",
+        "features": {
+          "color": true,
+          "chain": false,
+          "matrix": true,
+          "infrared": false,
+          "multizone": false,
+          "temperature_range": [
+            1500,
+            9000
+          ]
+        },
+        "upgrades": []
+      },
+      {
+        "pid": 187,
+        "name": "LIFX Candle Color US",
+        "features": {
+          "color": true,
+          "chain": false,
+          "matrix": false,
+          "infrared": false,
+          "multizone": false,
+          "temperature_range": [
+            1500,
+            9000
+          ]
+        },
+        "upgrades": []
+      },
+      {
+        "pid": 188,
+        "name": "LIFX Candle Color Intl",
+        "features": {
+          "color": true,
+          "chain": false,
+          "matrix": false,
+          "infrared": false,
+          "multizone": false,
+          "temperature_range": [
+            1500,
+            9000
+          ]
+        },
+        "upgrades": []
+      },
+      {
+        "pid": 203,
+        "name": "LIFX String US",
+        "features": {
+          "color": true,
+          "chain": false,
+          "matrix": false,
+          "infrared": false,
+          "multizone": true,
+          "temperature_range": [
+            1500,
+            9000
+          ],
+          "extended_multizone": true
+        },
+        "upgrades": []
+      },
+      {
+        "pid": 204,
+        "name": "LIFX String Intl",
+        "features": {
+          "color": true,
+          "chain": false,
+          "matrix": false,
+          "infrared": false,
+          "multizone": true,
+          "temperature_range": [
+            1500,
+            9000
+          ],
+          "extended_multizone": true
+        },
+        "upgrades": []
+      },
+      {
+        "pid": 205,
+        "name": "LIFX Outdoor Neon US",
+        "features": {
+          "color": true,
+          "chain": false,
+          "matrix": false,
+          "infrared": false,
+          "multizone": true,
+          "temperature_range": [
+            1500,
+            9000
+          ],
+          "extended_multizone": true
+        },
+        "upgrades": []
+      },
+      {
+        "pid": 206,
+        "name": "LIFX Outdoor Neon Intl",
```

Comment:
Update to indoor
#define LIFX_PRODUCT_ID_LCM4_INDOOR_NEON_SKIPPY_INTL    (0x00CE)   // SSID: LIFX Neon 123456 (Multi Zone Indoor Neon, INTL)